base/shell/cmd/cmd.h

```diff
@@ -272,6 +272,7 @@ BOOL   IsExistingFile (LPCTSTR);
 BOOL   IsExistingDirectory (LPCTSTR);
 BOOL   FileGetString (HANDLE, LPTSTR, INT);
 VOID   GetPathCase(TCHAR *, TCHAR *);
+void unquote(LPTSTR psz);
```

Comment:
And what is that function "StripQuotes" that exists just above?

Reply:
![VirtualBox_ReactOS_15_11_2019_10_28_32](https://user-images.githubusercontent.com/2107452/68909768-c9c56d00-0792-11ea-8ee2-4a8e18f616d9.png)
`StripQuotes` won't fix these problems.